length_test.go

```diff
@@ -399,7 +399,7 @@ func TestMsgCompressLengthLargeRecordsAllValues(t *testing.T) {
 	msg := new(Msg)
 	msg.Compress = true
 	msg.SetQuestion("redis.service.consul.", TypeSRV)
-	for i := 0; i < 900; i++ {
+	for i := 0; i < 90; i++ { // this was 900, for unknown reasons, brought down to 90 - questioning the value of the test in general. (for 900 this takes 2.41s)
```

Comment:
I think this was 900 for a good reason, git blame should point out what the PR was. I think compression was failing at specific byte boundaries and the high number was about catching that. It might be possible to raise the minimum instead. Put in the 900*2 records and then only compression for higher values.

Reply:
Perhaps leave this test for a separate PR.

Reply:
this is the top time taker (2.9s on my machine) - but yeah, prolly easier to discuss in another pr